zuul-core/src/main/java/com/netflix/zuul/netty/server/Server.java

```diff
@@ -203,6 +203,19 @@ public void start()
                 addressesToChannels.put(requestedNamedAddr.withNewSocket(chan.localAddress()), chan);
                 allBindFutures.add(nettyServerFuture);
             }
+
+            // All channels should share a single ByteBufAllocator instance.
+            // Add metrics to monitor that allocator's memory usage.
+            if (!allBindFutures.isEmpty()) {
```

Comment:
Should this conditional be inverted?

Reply:
We want to check here that the `allbindFutures` list is **not** empty because in line 201 we fetch the first entry. The conditional looks correct to me.